### DISCUSSION TITLE
Specfem2D_wave propagation

### DISCUSSION THREAD

**Discussion by fardy19** [community/community#62925] [Programming Help]
### Select Topic Area

Question

### Body

Please , Can anyone help me with the modeling of ultrasonic wave propagation in mortar using specfem2d? I am still a beginner and I'm really unsure where to start.
I would be very grateful to you.

**Comment by mehulrajdev** [comment]
Please support this. I can get non-experienced git devs to use fixup, but not to rebase locally. This would greatly help with PR workflows.
